### PR TITLE
Added forks to unified jobs table.

### PR DESCRIPTION
**Pull request by djyasin**
##### SUMMARY

Added job forks information to unified jobs table in analytic collector

##### ISSUE TYPE

 - New or Enhanced Feature

##### COMPONENT NAME

 - API

##### AWX VERSION

```
awx: 3.3.2
```

### CODE REVIEW

**Comment by djyasin** [comment]
Added forks column to unified jobs table collector. Here is a preview of the CSV file that gets generated:
```
id,polymorphic_ctype_id,model,organization_id,organization_name,execution_environment_image,inventory_id,inventory_name,created,name,unified_job_template_id,launch_type,schedule_id,execution_node,controller_node,cancel_flag,status,failed,started,finished,elapsed,job_explanation,instance_group_id,installed_collections,ansible_version,forks
1,11,workflowjob,1,Default,,,,2022-06-30 15:43:06.071451+00,Cats,8,manual,,"","",f,successful,f,2022-06-30 15:43:06.39069+00,2022-06-30 15:44:15.850194+00,69.460,"",,{},"",
2,9,job,1,Default,quay.io/ansible/awx-ee:latest,1,Demo Inventory,2022-06-30 15:43:06.691136+00,Demo Job Template,7,workflow,,receptor-1,awx_1,f,successful,f,2022-06-30 15:43:07.227155+00,2022-06-30 15:44:15.477331+00,68.250,"",2,"{""/usr/share/ansible/collections/ansible_collections"": {""awx.awx"": {""version"": ""21.2.0""}, ""amazon.aws"": {""version"": ""4.0.0""}, ""ovirt.ovirt"": {""version"": ""2.1.0""}, ""google.cloud"": {""version"": ""1.0.2""}, ""ansible.posix"": {""version"": ""1.4.0""}, ""ansible.windows"": {""version"": ""1.10.0""}, ""kubernetes.core"": {""version"": ""2.3.2""}, ""openstack.cloud"": {""version"": ""1.8.0""}, ""community.vmware"": {""version"": ""2.6.0""}, ""azure.azcollection"": {""version"": ""1.13.0""}, ""theforeman.foreman"": {""version"": ""3.4.0""}, ""redhatinsights.insights"": {""version"": ""1.0.7""}}}","ansible [core 2.12.5.post0]
",0
4,9,job,1,Default,quay.io/ansible/awx-ee:latest,1,Demo Inventory,2022-06-30 18:07:21.799975+00,Demo Job Template,7,manual,,receptor-1,awx_1,f,successful,f,2022-06-30 18:07:22.212381+00,2022-06-30 18:07:26.831799+00,4.619,"",2,"{""/usr/share/ansible/collections/ansible_collections"": {""awx.awx"": {""version"": ""21.2.0""}, ""amazon.aws"": {""version"": ""4.0.0""}, ""ovirt.ovirt"": {""version"": ""2.1.0""}, ""google.cloud"": {""version"": ""1.0.2""}, ""ansible.posix"": {""version"": ""1.4.0""}, ""ansible.windows"": {""version"": ""1.10.0""}, ""kubernetes.core"": {""version"": ""2.3.2""}, ""openstack.cloud"": {""version"": ""1.8.0""}, ""community.vmware"": {""version"": ""2.6.0""}, ""azure.azcollection"": {""version"": ""1.13.0""}, ""theforeman.foreman"": {""version"": ""3.4.0""}, ""redhatinsights.insights"": {""version"": ""1.0.7""}}}","ansible [core 2.12.5.post0]
",18
```

**Comment by djyasin** [comment]
Include forks on job

**Review by jbradberry — APPROVED** [review]

**Review by jbradberry — CHANGES_REQUESTED** [review]
Oh, I forgot: if you make a change to an existing collector, you need to bump the version number of the collector.

File: awx/main/analytics/collectors.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def events_table_partitioned_modified(since, full_path, until, **kwargs):
     return _events_table(since, full_path, until, 'main_jobevent', 'modified', project_job_created=True, **kwargs)
 
 
-@register('unified_jobs_table', '1.3', format='csv', description=_('Data on jobs run'), expensive=four_hour_slicing)
+@register('unified_jobs_table', '1.4', format='csv', description=_('Data on jobs run'), expensive=four_hour_slicing)
```
Comment (sarabrajsingh) [inline awx/main/analytics/collectors.py:399]: thanks for mentioning the version number bump

**Review by sarabrajsingh — APPROVED** [review]

**Review by TheRealHaoLiu — APPROVED** [review]

**Comment by djyasin** [comment]
> Oh, I forgot: if you make a change to an existing collector, you need to bump the version number of the collector.

This has been resolved!

**Review by jbradberry — APPROVED** [review]

**Review by CFSNM — APPROVED** [review]

**Pull request merged**